File: do.py
from odin.actions import action_map
from odin.utils.default import default_arguments_and_behavior

if __name__ == "__main__":
    args, model_wrapper = default_arguments_and_behavior()

    print("----SETUP----")
    print(args)
    print(model_wrapper)
    print("----END-SETUP----")

    action_map[args.action](model_wrapper, args=args)

File: do.jl
#=
do:
- Julia version: 
- Author: anderscs
- Date: 2018-11-07
=#
using PyCall

pushfirst!(PyVector(pyimport("sys")["path"]), "")

models = pyimport("odin.models")
def = pyimport("odin.utils.default")
actions = pyimport("odin.actions")

model_wrapper = models.load_model("pix2pix")
#args, model_wrapper = def.default_chainer()

actions.actions["train_model"]()


